aea/test_tools/click_testing.py

```diff
@@ -179,12 +179,12 @@ def teardown_class(cls) -> None:
     def setup(self) -> None:
         """Setup test."""
 
-        self.t = Path(tempfile.mkdtemp())
+        self.t = Path(tempfile.mkdtemp()).resolve()
```

Comment:
should fix this issue otherwise seen on macos: 
```
def test_teardown_and_teardown_cls(self) -> None:
        """Test teardown and teardown_class"""
    
        test_instance = self.setup_test()
        cwd = Path.cwd()
    
        assert not test_instance.t == cwd
        os.chdir(test_instance.t)
>       assert test_instance.t == Path.cwd()
E       AssertionError: assert PosixPath('/v.../tmpcxli7pxa') == PosixPath('/p.../tmpcxli7pxa')
E         +PosixPath('/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpcxli7pxa')
E         -PosixPath('/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpcxli7pxa')
```